src/main/java/com/moviesuggestion/controller/MovieController.java

```diff
@@ -0,0 +1,34 @@
+package com.moviesuggestion.controller;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import com.moviesuggestion.model.Movie;
+
+
+@RestController
+@RequestMapping("api/v1/")
+public class MovieController {
+
+    @RequestMapping(value = "movies", method = RequestMethod.GET)
```

Comment:
I know that "movies" comes intuitively, but usually in these cases the singular of the noun is used, i.e. "movie" because later when you will try to GET a single movie with a particular ID it will look weird GET movies/3 where in fact you are getting one movie with id 3.